tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "useDefineForClassFields": false, // ensure correct behaviour of class fields with decorators
 
     "strict": true, // for the full TypeScript experience
+    "strictPropertyInitialization": false, // to enable generic constructors, e.g. on CircuitValue
```

Comment:
if we go with this PR, this should be added in the zkapp CLI template projects as well

Reply:
I agree we should fix this globally in the CLI and push users to do the same instead fixing locally by adding a `!` to the prop, e.g. `@prop value!: Field`.